src/main/java/org/spongepowered/api/item/inventory/type/Interactable.java

```diff
@@ -24,23 +24,20 @@
  */
 package org.spongepowered.api.item.inventory.type;
 
-import org.spongepowered.api.entity.Entity;
+import org.spongepowered.api.entity.living.player.Player;
 import org.spongepowered.api.item.inventory.Inventory;
 
 /**
- * Interface for inventories which may be interacted with by specific types of
- * Entity.
- *
- * @param <T> Base type of entity which may interact with this object  
+ * Interface for inventories which may be interacted with by Players.
  */
-public interface Interactable<T extends Entity> extends Inventory {
```

Comment:
No they really can't @kashike...You may say they are "interacting" with inventories, but there is no GUI being displayed, no data coming from any client, it can all be done with simple inventory transactions. As I said these methods should not be shared by Humanoid and Player because they do not mean the same thing in both of those contexts.